documentation/sphinx/source/metrics.rst

```diff
@@ -0,0 +1,209 @@
+.. _metrics:
+
+######################################################
+Metrics
+######################################################
+
+.. _metric-introduction:
+
+Introduction
+============
+
+FoundationDB's has different metrics.
+
+### ProcessMetrics
+
+ProcessMetrics are responsible for getting information of events listed:
+
+- Current Connections: Data count connection established
+- Connection Established: Number of connection count established per elapsed minutes.
+- Connection Closed: Number of open connections closed
+- Connection Errors: Number of error received in an open connection
+- Memory: Memory size
+- Unsued Allocated Memory: Size of the unused allocated memory
+- Memory Limit: Max memory limit
+- Disk Total Bytes: Total number of bytes present in disk
+- Disk Free Bytes: Total number of free bytes present in disk
+- Cache Hits: Number of cache hits
+- Cache Misses: Number of cache misses
+- Elapsed: Elapsed time
+- Machine IP: Machine address
+- ID: Unique ID
+- CPU Seconds: CPU Seconds available
+- MainThreadCPUSeconds: CPU seconds for main thread of the process
+- UptimeSeconds: Time units for up time of a process
+- ResidentMemory: Resident memory size is the portion of memory occupied by the process
+  held by the random access memory.
+- AllocatedMemory [16-4096]: Allocated memory for from range 16 - 4096 bytes size.
+- ApproximateUnusedMemory [16-4096]:  Unused memory from range 16 - 4096 bytes size.
+- MbpsSent: Mega bytes per second sent.
```

Comment:
Looking at code over here[1] and the notation (Mbps vs MBps) it should be "megabits per second". Also can be more accurately descriptive by saying something like "Network bandwidth for outgoing data in megabits/second".

[1] https://github.com/apple/foundationdb/blob/master/flow/SystemMonitor.cpp#L70